handler/src/main/java/io/netty/handler/timeout/IdleStateHandler.java

```diff
@@ -34,6 +35,9 @@
  * Triggers an {@link IdleStateEvent} when a {@link Channel} has not performed
  * read, write, or both operation for a while.
  *
+ * The {@link IdleStateHandler} should be placed as first {@link ChannelHandler}
```

Comment:
Sure... Say you use HttpServerCodec.

An attacker can open a connection and very slowly trickle bytes of a valid HTTP request and just staying below the IdleStateHandler's threshold. Allowing the attacker to open many connections at a relative low bandwidth cost.